pkg/cableengine/cableengine_test.go

```diff
@@ -146,51 +151,6 @@ var _ = Describe("Cable Engine", func() {
 					fakeDriver.AwaitDisconnectFromEndpoint(&prevEndpoint)
 					fakeDriver.AwaitConnectToEndpoint(remoteEndpoint)
 				})
-
```

Comment:
All those go away sadly because the operation becomes async and we wait for the NAT discovery to return the nat details to setup the connection.

Reply:
I think we can revive these when we have retries with a workqueue.